docs/index.md

```diff
@@ -337,25 +337,35 @@ The following limitations apply when presenting Health Card as QR codes, rather
   * Verifier cannot include purposes of use in-band
 * Does not capture a digital record of the presentation
 
-When a Health Card JWS exceeds 1191 characters, it must be split into multiple QR Codes to ensure readability. Each "chunk" of 1191 characters is numerically encoded and prefixed with an ordinal as well as the total number of chunks required to re-assemble the JWS. 
 
-## Multi-part QR codes
+## Creating a QR code (or a set of QR codes) from a Health Card JWS
 
+### Chunking
 
-### Numerical Encoding
+Commonly, Health Cards will fit in a single V22 QR code.  Any JWS longer than 1195 characters is split into "chunks" of length 1191 or smaller; each chunk is then encoded as a separate QR code to ensure ease of scanning.  Each chunk is numerically encoded and prefixed with an ordinal as well as the total number of chunks required to re-assemble the JWS, as described below.
```

Comment:
```suggestion
Commonly, Health Cards will fit in a single V22 QR code.  Any JWS longer than 1191 characters is split into "chunks" of length 1191 or smaller; each chunk is then encoded as a separate QR code to ensure ease of scanning.  Each chunk is numerically encoded and prefixed with an ordinal as well as the total number of chunks required to re-assemble the JWS, as described below.
```

Josh - this was just a holdover, right?

Reply:
Ohhh ... no, this was intentional ... b/c the removal of the chunking data gives us those four characters. Got it.